about.css

```diff
@@ -0,0 +1,515 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
```

Comment:
- [ ] I kindly would suggest you improve the structure of your About page by modeling it according to the template design provided. All page elements should have similar spacing, borders, shadows, and paddings to the original design. Take a look at the original design and adjust your design accordingly.

> Template design
<img width="687" alt="Screenshot 2023-04-27 at 14 35 24" src="https://user-images.githubusercontent.com/71140133/234887430-19d721be-56e5-4130-b2be-3c60f3f8c3b5.png">

> Your design
<img width="1142" alt="Screenshot 2023-04-27 at 14 36 01" src="https://user-images.githubusercontent.com/71140133/234887736-5743c2ee-da99-4fea-9fcc-8a26bd20f48e.png">